src/model/BrandModel.ts

```diff
@@ -0,0 +1,37 @@
+import { getPostgresClient } from "@/services/server/database";
+import { Brand } from "@/services/types";
+
+async function getBrand(brandId: string): Promise<Brand | undefined> {
```

Comment:
Shouldn't `brandId` be `number`? You modified it to `INT` in this commit, right? Talking about the SQL column.